README.md

```diff
@@ -97,6 +97,24 @@ public class MainApplication extends Application implements ReactApplication {
 }
 ```
 
+```kotlin
+// react-native-splash-screen >= 0.3.1
+import org.devio.rn.splashscreen.SplashScreen
+
+
+class MainActivity : ReactActivity() {
+
+    override fun getMainComponentName(): String = "main"
+    override fun onCreate(savedInstanceState: Bundle?) {
+        SplashScreen.show(this)
+        setTheme(R.style.AppTheme);
+        super.onCreate(null)
```

Comment:
Hello,
Why was `setTheme(R.style.AppTheme);` added?
I'm asking because I use it without it, and it works as expected.
Thanks
```suggestion
        SplashScreen.show(this)
        super.onCreate(savedInstanceState);
```
Thanks